src/logging.interceptor.ts

```diff
@@ -19,8 +19,6 @@ export class LogsMiddleware implements NestMiddleware {
       if (statusCode >= 400) {
         return this.logger.warn(message);
       }
-
-      return this.logger.log(message);
```

Comment:
NIT: Ideally, this would be configurable by setting a `LOG_LEVEL` env var or similar. Not sure how this works in NestJS though. Something we can look into later.